frontend/src/scripts/index.js

```diff
@@ -0,0 +1,91 @@
+const loginForm = document.getElementById('login-form-action')
+const registerForm = document.getElementById('register-form-action')
+
+function showLoginScreen() {
+  document.getElementById('login-form').style.display = 'flex'
+  document.getElementById('register-form').style.display = 'none'
+}
+
+function showRegisterScreen() {
+  document.getElementById('login-form').style.display = 'none'
+  document.getElementById('register-form').style.display = 'flex'
+}
+
+document.getElementById('show-register').addEventListener('click', function () {
+  showRegisterScreen()
+})
+
+document.getElementById('show-login').addEventListener('click', function () {
+  showLoginScreen()
+})
+
+showLoginScreen()
+
+loginForm.addEventListener('submit', async (e) => {
+  e.preventDefault()
+  email = e.target.name.value
```

Comment:
```suggestion
  email = e.target.email.value
```
Pegando o valor do e-mail pelo atributo "name"